includes/settings/class-wcpdf-settings-callbacks.php

```diff
@@ -714,6 +717,23 @@ public function validate( $input ) {
 		// Return the array processing any additional functions filtered by this action.
 		return apply_filters( 'wpo_wcpdf_validate_input', $output, $input );
 	}
+
+	/**
+	 * Appends language at the end of the setting provided, in case the setting is translatable
+	 * and it does not have a language set.
+	 *
+	 * @param string $setting Settings field that needs a language.
+	 * @param array  $args Setting arguments.
+	 *
+	 * @return void
+	 */
+	public function append_language( string $setting, array $args ): string {
+		if ( isset( $args['translatable'] ) && isset( $args['lang'] ) && 'default' !== $args['lang'] && true == $args['translatable'] && ! ( substr( $setting, -strlen( "[{$args['lang']}]" ) ) === "[{$args['lang']}]" ) ) {
```

Comment:
Maybe `true === $args['translatable']`?

Reply:
@alexmigf I left == because sometimes, `$args['translatable']` comes as a string `'true'`, and sometimes as a boolean. With ==, both cases are covered.

Reply:
Just checking in—did you notice it uses `===` over [here](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/e6496688b9484860f232eae1ce4fab90c4380cc0/includes/settings/class-wcpdf-settings-callbacks.php#L201)?

Reply:
@alexmigf I did not notice that, interesting. I am not sure why sometimes it returns argument as a string and sometimes as a bool. But nevertheless, I think that true == should be OK for this case. What do you think?

Reply:
I think you need to find why that is happening.

Reply:
@alexmigf this is happening because PHP decodes bool values as strings via AJAX. We can solve this by checking if the translation is string and if yes, convert it to boolean, however, I think that == true is the best possible solution in this case because it will check if translation is true or "true", and both of those indicate that we want to translate the setting.
Do you have an alternative solution?

Reply:
Yes, you could check if the value is a `string` and apply this function `wc_string_to_bool()`, otherwise just deal with the `boolean`.